home/templates/includes/navigation.html

```diff
@@ -188,71 +188,55 @@
           <li class="nav-item">
             <a href="{% url 'about_us' %}" class="nav-link"> About </a>
           </li>
-          <li class="nav-item dropdown">
+          <li class="nav-item dropdown1">
             <a
               href="#"
               class="nav-link dropdown-toggle"
-              id="frontPagesDropdown"
+              id="projectsDropdown"
               aria-expanded="false"
-              data-bs-toggle="dropdown"

```

Comment:
Please add " data-bs-toggle="dropdown"" to both Project and Blog, as the dropdown is not functioning

Reply:
Hi, I have used custom javascript to handle the dropdown, so there is no need to add these attributes, dropdown is working fine on my end, let me know if you are facing any problem.
<img width="269" alt="Screenshot 2024-12-07 235332" src="https://github.com/user-attachments/assets/9768ceb3-a826-4026-8bea-e4365d5f594a">

Reply:
Hey @NaeemTufail0 , using your work, here's the video for evidence:

https://github.com/user-attachments/assets/af4418b6-fc57-42a4-ae0a-a27a8ac50a6d

Reply:
Hey @jbm1412 , As you can see the background of my SS was white and your was yellow. May be you are on any other branch. I checked on my end everything is working fine as i attached SS above.

Reply:
Hey @NaeemTufail0 , the issue was from my side due to outdated and didn't clear my cache. It's working now!! Sorry for the inconvenice

Reply:
> Hey @NaeemTufail0 , the issue was from my side due to outdated and didn't clear my cache. It's working now!! Sorry for the inconvenice

All good, Thanks